app/uk/gov/hmrc/accessibilitystatementfrontend/models/TypeOfService.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.accessibilitystatementfrontend.models
+
+import io.circe.{Decoder, Encoder}
+
+sealed trait TypeOfService
```

Comment:
feels like we could reduce a lot of the repetition here by introducing some traits for these enums?  something like
```
trait EnumValue {
  def value: String
  override def toString: String = value
}

trait Enum[T <: EnumValue] {
  def description: String
  def values: Seq[T]

  implicit val encoder: Encoder[T] = Encoder.encodeString.contramap[T](_.toString)
  implicit val decoder: Decoder[T] = Decoder.decodeString.emap { value =>
    values.find(_.value == value)
      .map(Right.apply)
      .getOrElse(Left(s"""Unrecognised $description "$value""""))
  }
}

sealed trait LiveOrClassic extends EnumValue

object LiveOrClassic extends Enum[LiveOrClassic] {
  def description: String = "live or classic service"
  def values: Seq[LiveOrClassic] = Seq(ClassicServices, LiveServicesEdinburgh, LiveServicesNewcastle, LiveServicesTelford, LiveServicesWorthing)
}

case object ClassicServices extends LiveOrClassic {
  val value = "Classic Services"
}

...
```